tispark/tispark-user-guide.md

```diff
@@ -239,6 +194,58 @@ TiSparkR is a thin layer built to support the R language with TiSpark. Refer to
 
 TiSpark on PySpark is a Python package build to support the Python language with TiSpark. Refer to [this document](https://github.com/pingcap/tispark/blob/master/python/README.md) for usage.
 
+## Use TiSpark together with Hive
+TiSpark should be ok to use together with Hive. 
+You need to set environment variable HADOOP_CONF_DIR to your Hadoop's configuration folder and copy hive-site.xml to spark/conf folder before starting Spark.
+```
+val tisparkDF = spark.sql("select * from tispark_table").toDF
+tisparkDF.write.saveAsTable("hive_table") // save table to hive
+spark.sql("select * from hive_table a, tispark_table b where a.col1 = b.col1").show // join table across Hive and Tispark
+```
+
+## Load Spark Dataframe into TiDB using JDBC
+TiSpark does not provide a direct way to load data into yout TiDB cluster, but you can still load using jdbc like this:
+```scala
```

Comment:
Add a blank line before the code block, to avoid possible display error on the PingCAP website.